robot/Cumulus/resources/locators_50.py

```diff
@@ -311,7 +313,14 @@
         'alert':'//div[@role="alert" and contains(@class,"slds-theme_{}")][./span[text()="{}"]]/h2[text()="{}"]',
         'element_text':'//*[contains(@data-qa-locator,"{}") and text()="{}"]',
         'page_error':'//div[@class="pageLevelErrors"]',
-        'picklist_values':'//*[contains(@data-qa-locator,"{}")]//child::lightning-base-combobox-item[@data-value="{}"]'
+        'picklist_values':'//*[contains(@data-qa-locator,"{}")]//child::lightning-base-combobox-item[@data-value="{}"]',
+        'modal_field':'//div[@class="modal-body scrollable slds-modal__content slds-p-around--medium"]//*[contains(@data-qa-locator,"{}")]//child::{}',
+        'modal_button':'//div[@class="modal-body scrollable slds-modal__content slds-p-around--medium"]//*[contains(@data-qa-locator,"{}")]/button',
+        'modal_id':'//div[@class="modal-body scrollable slds-modal__content slds-p-around--medium"]//*[contains(@data-qa-locator,"{}")]',
+        'modallookup-option':'//div[@class="modal-body scrollable slds-modal__content slds-p-around--medium"]//li/lightning-base-combobox-item[.//*[@title="{}"]]',
+        'remove_lookup':'//div[@class="modal-body scrollable slds-modal__content slds-p-around--medium"]//*[contains(@data-qa-locator,"button Remove selected option Opportunity: Record Type")]',
+        'click-dropkown-value':"//div[@class='modal-body scrollable slds-modal__content slds-p-around--medium']//*[contains(@class,'slds-combobox')]//span[text()='{}'']"
```

Comment:
It is better to avoid class values like 'scrollable', 'slds-p-around--medium', because they related to css styling of the page. If instead of 'medium' size, the styling changes to 'large', your locator will fail. Something like this will be good:
```
'modal_field':'//div[contains(@class,"modal-body")]//*[contains(@data-qa-locator,"{}")]//child::{}',
'modal_button':'//div[contains(@class,"modal-body")]//*[contains(@data-qa-locator,"{}")]/button',
```

Reply:
got it, thanks.